.github/workflows/Pipeline.yml

```diff
@@ -421,14 +430,14 @@ jobs:
     name: 🗑️ Artifact Cleanup
     runs-on: ubuntu-latest
     needs:
-      - Package
-      - PublishOnPyPI
+      - Coverage
```

Comment:
Coverage, StaticTypeCheck and BuildTheDocs are dependencies of PublishToGitHubPages. Therefore, it is redundant to have them added here.

Reply:
According to the documentation, a job must be specified in `needs` if output variables are used.

From a technical point is makes sense. I don't understand why it works without.

I specified the jobs creating the artifact and the last job we wait for to destroy them.
While this creates more edges, it's more robust for restructuring job dependencies.

Reply:
Gotcha.